SingularityService/src/main/java/com/hubspot/singularity/data/InactiveSlaveManager.java

```diff
@@ -0,0 +1,43 @@
+package com.hubspot.singularity.data;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.apache.curator.framework.CuratorFramework;
+
+import com.codahale.metrics.MetricRegistry;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.hubspot.singularity.config.SingularityConfiguration;
+
+@Singleton
+public class InactiveSlaveManager extends CuratorManager {
+  private static final String ROOT_PATH = "/inactive";
```

Comment:
nit pick here, but could we have this be `/inactiveSlaves`? when looking at the top level in zk would be easier to tell what it's for